src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,58 @@
+.main-content {
+  height: 100vh;
+  
+  @include flexCenter;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  width: 726px;
+  padding: 10px;
+
+  @for $numberOfDays from 29 through 31 {
+    &--month-length--#{$numberOfDays} > :nth-child(n + #{$numberOfDays + 1}) {
+      display: none;
+    }
+  }
+
+  $margins:
+    "mon" 0,
+    "tue" $cell-size + $gap,
+    "wed" ($cell-size + $gap) * 2,
+    "thu" ($cell-size + $gap) * 3,
+    "fri" ($cell-size + $gap) * 4,
+    "sat" ($cell-size + $gap) * 5,
+    "sun" ($cell-size + $gap) * 6;
+
+  @each $day, $margin in $margins {
+    &--start-day--#{$day} > :first-child {
+      margin-left: $margin;
+    }
+  }
```

Comment:
Duplicate code "($cell-size + $gap)". It can be moved into @each loop.

Reply:
agree